src/main/java/net/minecraftforge/client/settings/KeyMappingLookup.java

```diff
@@ -59,6 +60,16 @@ private KeyMapping get(InputConstants.Key keyCode, KeyModifier keyModifier)
         return null;
     }
 
+    /**
+     * Returns all active keys associated with the given key code and the active
+     * modifiers and conflict context.
+     *
+     * This could probably be renamed to something like getActiveMappings, but
+     * that would be a breaking change.
```

Comment:
We could perhaps rename this method to `getActive` (I think the `Mappings` part is redundant in a class named `KeyMappingLookup`) and add a `getAll` bouncer to the newly-renamed method (which is deprecated and documented in the same manner as `get` above). I'll leave that to your discretion.